_includes/list_project_team.html

```diff
@@ -1,20 +1,42 @@
+
+{%- if include.inline -%}
+{%- assign li = "" -%}
+{%- assign eli = "" -%}
+{%- else -%}
+{%- assign li = "<li>" -%}
+{%- assign eli = "</li>" -%}
 <h3>Team</h3>
 <ul>
-{%- for member in include.team -%}
-  {%- assign found=false -%}
-  {%- for person_hash in site.data.people -%}
-    {%- assign person = person_hash[1] -%}
-    {%- if person.shortname == member -%}
-      {%- assign found=true -%}
-      {%- if person.website -%}
-        <li> <a href="{{person.website}}">{{ person.name }}</a> </li>
-      {%- else -%}
-        <li> {{ person.name }} </li>
-      {%- endif -%}
-    {%- endif -%}
-  {%- endfor -%}
-  {%- unless found -%}
-    <li> {{ member }} </li>
-  {%- endunless -%}
-{%- endfor -%}
+{%- endif -%}
+
+{% for contact in include.team %}
+  {% assign written = false %}
+  {% if contact contains "](" %}
+  {% elsif contact contains "@" %}
+    <a href="mailto:{{contact}}"> <em>{{contact}}</em> </a>
+    {% assign written = true %}
+  {% else %}
+    {% for person_hash in site.data.people -%}
+      {% assign person = person_hash[1] -%}
+      {% if person.shortname == contact %}
+        {%- if inline and person.e-mail -%}
```

Comment:
Slightly hacky - email is included and prioritized only in the "inline" form. Can be moved to a new variable if there's a need. Current inline is for the fellows page only.